hyrax/config/blacklight.yml

```diff
@@ -2,7 +2,7 @@
 
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL_SCHEME'] || 'http' %>://<%= ENV['SOLR_HOST'] || '127.0.0.1' %>:<%= ENV['SOLR_PORT'] || '8983' %>/solr/hydra-development
+  url: <%= ENV['SOLR_URL_SCHEME'] || 'http' %>://<%= ENV['SOLR_HOST'] || '127.0.0.1' %>:<%= ENV['SOLR_PORT'] || '8983' %>/solr/hyrax_production
```

Comment:
we want to be able to run in dev mode and not prod mode locally - however it didn't make sense to do all the work to change the solr name and make it a variable everywhere.